src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -46,19 +32,17 @@ namespace stan {
       }
 
       void set_max_delta(double d) {
-        max_delta_ = d;
+        max_deltaH_ = d;
       }
 
       int get_max_depth() { return this->max_depth_; }
-      double get_max_delta() { return this->max_delta_; }
+      double get_max_delta() { return this->max_deltaH_; }
 
       sample transition(sample& init_sample,
                         interface_callbacks::writer::base_writer& writer) {
         // Initialize the algorithm
         this->sample_stepsize();
 
-        nuts_util util;
-
         this->seed(init_sample.cont_params());
```

Comment:
Thanks for the pointer.  I thought it was some awful C++ thing like that.

> On Apr 8, 2016, at 12:13 PM, Michael Betancourt <notifications@github.com> wrote:
> 
> In src/stan/mcmc/hmc/nuts/base_nuts.hpp:
> 
> >  
> >        sample transition(sample& init_sample,
> >                          interface_callbacks::writer::base_writer& writer) {
> >          // Initialize the algorithm
> >          this->sample_stepsize();
> >  
> > -        nuts_util util;
> > -
> >          this->seed(init_sample.cont_params());
> 
> It's a template issue -- lookup occurs only after template instantiation so the internal class members and methods aren't in scope. See http://stackoverflow.com/questions/4643074/why-do-i-have-to-access-template-base-class-members-through-the-this-pointer.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>